buildpipeline/Dotnet-CoreClr-Trusted-BuildTests.json

```diff
@@ -343,7 +343,7 @@
       "value": ""
     },
     "AgentToolsPath": {
-      "value": "$(Agent.HomeDirectory)\\AgentTools\\"
+      "value": "$(AgentToolsPath)"
```

Comment:
Does this cause recursive value issue? Perhaps just remove the AgenToolsPath variable declaration completely.

Reply:
I don't think it would but yeah, one less thing to change